Lib/tkinter/tix.py

```diff
@@ -63,6 +65,10 @@
 TCL_IDLE_EVENTS   = 1 << 5
 TCL_ALL_EVENTS    = 0
 
+# Deprecation warning
+warnings.warn('the tkinter.tix module is deprecated in favour of tkinter.ttk and is set \
+to be removed in the near-future', DeprecationWarning, stacklevel=2)
```

Comment:
```suggestion
warnings.warn('the tkinter.tix module is deprecated in favour of tkinter.ttk and is set '
              'to be removed in the near-future', DeprecationWarning, stacklevel=2)
```
I think this looks clearer this way